### DISCUSSION TITLE
DB Navigator does not show functions

### DISCUSSION THREAD

**Discussion by kaelte** [dbeaver/dbeaver#12137] [Q&A] [answered]
* Ubuntu 20.04.2 LTS
* DBeaver Version 21.0.2.202104042040
* PostgreSQL 12.6 (Ubuntu 12.6-0ubuntu0.20.04.1)

DB Navigator does not show functions anymore, cf. screenshot.
But the query in SQL editor returns many functions defined in schema adressen.
```
select * from information_schema.routines
where specific_schema='adressen' order by routine_name;
```
![dbeaver_functionsNotShown](https://user-images.githubusercontent.com/10630145/115146749-24ce5400-a058-11eb-9b57-12a36c36293a.png)

Moreover table and views are not separated in different branches in the navigator tree.

**Comment by uslss** [comment, marked as answer]
@kaelte , you should switch to Advanced view to see mentioned objects.
It could be done using Connection context menu:
![image](https://user-images.githubusercontent.com/31996417/115155968-d71f0f00-a08a-11eb-910e-4032e0083dfb.png)
Or in the Connection properties:
![image](https://user-images.githubusercontent.com/31996417/115155996-fae25500-a08a-11eb-98b1-ef53ea6bd04f.png)

**Comment by kaelte** [comment]
Thanks @uslss for your help!
That works.
